File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGUnit.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGUnit //Type originates from: YGEnums.h
{
    YGUnitUndefined,
    YGUnitPoint,
    YGUnitPercent,
    YGUnitAuto;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGUnit forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGUnit.kt
package app.cash.redwood.yoga.internal.enums

/** Type originates from: YGEnums.h */
enum class YGUnit {
  YGUnitUndefined,
  YGUnitPoint,
  YGUnitPercent,
  YGUnitAuto;

  fun getValue(): Int {
    return ordinal
  }

  companion object {
    fun forValue(value: Int): YGUnit {
      return values()[value]
    }
  }
}
